src/components/Blocking/SpecificIssue.js

```diff
@@ -110,7 +110,7 @@ const styles = StyleSheet.create({
     backgroundColor: COLORS.almostBlack
   },
   draggableIcon: {
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color for the draggable icon has been updated from `COLORS.lightgrey` to `COLORS.gray100`. Verify that this change is consistent with the rest of the application's design and doesn't affect the icon's visibility.
```diff
-    backgroundColor: COLORS.lightgrey
+    backgroundColor: COLORS.gray100
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->